docs/usage.rst

```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
 - :rfc:`9073` - Event Publishing Extensions to iCalendar
 - :rfc:`9253` - Support for iCalendar Relationships
 
-File structure
+iCalendar File structure
 --------------
 
-An iCalendar file is a text file (utf-8) with a special format. Basically it
-consists of content lines.
+An iCalendar file is a text file (utf-8) with a special format.
 
-Each content line defines a property that has 3 parts (name, parameters,
-values). Parameters are optional.
+It consists of **content lines**,
+with each content line defining a property that has 3 parts: name, parameters and value. Parameters are optional.
 
-A simple content line with only name and value could look like this::
+Example 1: a simple content line (with only name and value)::
 
   BEGIN:VCALENDAR
 
-A content line with parameters can look like this::
+Example 2: a content line with parameters::
 
   ATTENDEE;CN=Max Rasmussen;ROLE=REQ-PARTICIPANT:MAILTO:example@example.com
 
-And the parts are::
+The parts in this example are::
 
   Name:   ATTENDEE
   Params: CN=Max Rasmussen;ROLE=REQ-PARTICIPANT
   Value:  MAILTO:example@example.com
 
-Long content lines are usually "folded" to less than 75 character, but the
-package takes care of that.
+Note: Long content lines are usually "folded" to less than 75 characters (the
+package takes care of this).
 
+On a higher level, you can think of iCalendar files structured as having components and sub components.
 
-Overview
---------
+A component will have properties with values. The values
+have special types, like integer, text, and datetime. These values are
+encoded in a special text format in an iCalendar file. This package contains methods for converting to and from these encodings.
```

Comment:
I've never been a fan of arbitrary line wrapping in narrative documentation, but if we're going to do that, then we should be consistent.

Narrative documentation is not code, and we're no longer constrained by 80-character terminal sessions over dial-up modems when editing. To me, it makes more sense to have one sentence per line, eliminating the need for line wrapping and making for easier review of diffs, but I won't push for it.

Reply:
Agreed either way it should be consistent. Are you OK with creating a separate issue to discuss the line wrapping in the docs?